content/posts/2024-11-28_QChat_news.md

```diff
@@ -0,0 +1,94 @@
+---
+authors:
+  - gallaman
+categories:
+  - Announcements
+comments: true
+date: 2024-11-28
+description: "Some news about QChat and the features provided by the new 1.1 version. #GISChat #GISTribe"
+icon: material/chat
+license: beerware
+links:
+  - QTribu documentation: https://qtribu.geotribu.fr/
+  - Initial announcement: https://blog.geotribu.net/2024/10/18/introducing-qchat-a-chat-within-qgis/
```

Comment:
I think related hyperlink is better. See https://squidfunk.github.io/mkdocs-material/tutorials/blogs/basic/#related-links

Reply:
The CI build job seems to fail, after trying "absolute" and relative link

Reply:
weird. And locally?

Reply:
Locally, I don't get any error when building the website with the `--strict` option.
I'm using the `mkdocs-free.yml` config :thinking:, and the `Related links` are not present under the `Metadata` section in the left part of the page

Reply:
Ok @Guts, fixed :tada: 

Actually, the trick is that the external links "root" is the `content` folder, [this commit](https://github.com/geotribu/english-blog/pull/13/commits/feacaa5ddfcea9ddf96902a93a52e35705a6234a) solves it